content/docs/introduction/overview.md

```diff
@@ -33,6 +35,14 @@ Prometheus's main features are:
 * targets are discovered via service discovery or static configuration
 * multiple modes of graphing and dashboarding support
 
+### What are metrics ?
+
+In layperson terms, metrics are a standard for measurement. What we want to measure differs from application to application. For a web server it can be request times, for a database it can be number of active connections or number of active queries etc.
```

Comment:
```suggestion
In layperson terms, metrics are numeric measurements, time series mean that changes are recorded over time. What users want to measure differs from application to application. For a web server it might be request times, for a database it might be number of active connections or number of active queries etc.
```